vkbeautify/vkbeautify-tests.ts

```diff
@@ -0,0 +1,119 @@
+import VKBeautify from 'vkbeautify';
```

Comment:
I did, but now I'm getting TSLint errors, telling me to 

48:1  export-just-namespace  Instead of `export =`-ing a namespace, use the body of the namespace as the module body.

I've been puzzled by that message, and tried re-reading the TypeScript namespaces page but I still don't think I get it.

Is it suggesting to go

```
export namespace vkbeautify {
    ...
}
```
?